server/routes/stripe.js

```diff
@@ -2,16 +2,17 @@ const express = require("express");
 const router = express.Router();
 const {
   createSession,
-  createCustomer,
   retrieveCustomer,
-  createPaymentMethod,
+  retrieveAllCustomers,
   getAllPaymentMethods,
+  sendPayment,
 } = require("../controllers/stripe");
+const protect = require("../middleware/auth");
 
 router.route("/session").post(createSession);
-router.route("/customers/create").post(createCustomer);
 router.route("/customers/retrieve/:id").get(retrieveCustomer);
-router.route("/payment/create").post(createPaymentMethod);
-router.route("/payment/all").get(getAllPaymentMethods);
+router.route("/customers/retrieveAll").get(retrieveAllCustomers);
+router.route("/payment/all/:customerId").get(protect, getAllPaymentMethods);
```

Comment:
```suggestion
router.route("/payment/all").get(protect, getAllPaymentMethods);
```
We no longer require that path param 😄